lib/ember-test-helpers/test-context.js

```diff
@@ -7,3 +7,7 @@ export function setContext(context) {
 export function getContext() {
   return __test_context__;
 }
+
+export function unsetContext() {
```

Comment:
wanted it to be very obvious that It was specifically to unset, not just an overloaded the existing set.